conf/login_darkstar.conf

```diff
@@ -79,6 +79,30 @@ search_server_port: 54002
 
 expansions: 30
 
+#Account features - 2 Bytes?
+#
+#Byte 1
+#00000000 Bit0 - Secure Token Icon

```

Comment:
Fizz, you packet captured our server -- ignoring the work I put in to find the correct packet and bytes -- and claimed it as your own.  It would have been nice to let me submit the PR myself.  Please be more considerate of developers going forward.  Thank you.